src/redux/customer-resource.js

```diff
@@ -55,10 +58,15 @@ export const customerResourceEpics = combineEpics(
     deserialize: (payload) => {
       if (payload) {
         let { customerResourcesList, ...title } = payload;
+
         if (customerResourcesList[0] && customerResourcesList[0].contentType) {
           customerResourcesList[0].contentType = formatContentType(customerResourcesList[0].contentType);
         }
 
+        if (title.pubType) {
+          title.pubType = formatPublicationType(title.pubType);
```

Comment:
This seems more like a view level concern to me. Is there any reason we wouldn't actually format the publication type not inside an epic or a reducer, but in the component that displays it?

(I realize that this is where `formatContentType`, but perhaps we should move both.

Reply:
I think the idea is that this will be a job of `mod-kb-ebsco` very soon

Reply:
Oddly enough - We had started by making the changes for formatContentType in the view as you suggest as well. But the thought was that this will move to mod-kb-ebsco (as wil indicates) as the transformation rules/values will be specific to the resource management backend.   So the thought was to move it closer to the backend

Reply:
Why not just make them in mod-kb-ebsco now?

Reply:
It's just a passthrough for now, and we aren't handling any data yet. Once we start on the JSON-API conversion, this will be easy enough to plop in.

Reply:
OK, gotcha.